src/backend/tests/projects.test.ts

```diff
@@ -1168,17 +1181,45 @@ describe('Projects', () => {
         iconName: 'Doc2'
       });
 
-      const updated = await ProjectsService.editLinkType(mockLinkType1.name, 'Doc2', mockLinkType1.required, batman);
+      const updated = await ProjectsService.editLinkType(
+        mockLinkType1.name,
+        mockLinkType1.name,
+        'Doc2',
+        mockLinkType1.required,
+        batman
+      );
 
       expect(updated).toEqual(transformedMockLinkType1);
       expect(prisma.linkType.update).toHaveBeenCalledWith({
         where: { name: mockLinkType1.name },
         data: {
+          name: mockLinkType1.name,
           iconName: 'Doc2',
           required: mockLinkType1.required
         },
         ...linkTypeQueryArgs
       });
     });
+    test('LinkType edits successfully, changing name as well', async () => {
```

Comment:
we're doing tests a new way, and we wanna do all new tests in this way, see how Aaryan did it in this PR and lmk if u have questions: https://github.com/Northeastern-Electric-Racing/FinishLine/pull/2251 (you'll probably need to make a new test file for this)